docker/dev/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       context: ../../
       dockerfile: docker/dev/Dockerfile
     ports:
-      - "127.0.0.1:8536:8536"
+      - "8536:8536"
```

Comment:
This means that the service will be available on the local network. Do you really want that? Afaik it was working perfectly fine before.

Reply:
Ya, its for dev, and I have a different computer on my local running these builds now, so I needed these opened up.

Reply:
Hmm not sure if its a good idea to publish it like this, imagine if someone uses this on a public network. How about you put this in an override file and put that in gitignore? https://docs.docker.com/compose/extends/

Reply:
I don't think its a big deal, because these dev images are never published to dockerhub, they'd have to be built on someones machine, so they wouldn't / shouldn't be used for prod servers.

Reply:
I'm talking about someone who is developing with this setup on some kind of shared wifi or such. At least there should be a note about it in the docs imo.